WirtBot/src/managed_dns.rs

```diff
@@ -0,0 +1,56 @@
+use std::env;
+
+const MANAGED_DNS_FEATURE_FLAG :&str = "MANAGED_DNS_ENABLED";
```

Comment:
Nice one. Ill try to adapt the other flags to use consts like this after your PR as well!